mmcif/tests/testBinaryCifWriter.py

```diff
@@ -129,6 +134,18 @@ def testSerialize(self):
             logger.exception("Failing with %s", str(e))
             self.fail()
 
+    def __verifyEncoding(self, fname, storeStringsAsBytes):
+        """Verifies encoding"""
+        exe = sys.executable
+        args = [exe, os.path.join(HERE, "bcifprint.py"), fname]
```

Comment:
Why choose to call `bcifprint.py` as an executable file, rather than importing the class and calling some method? Is it because you want to be able to use that script for other cases of testing?

Reply:
I have left the script around as a standalone - but modified it to suppress output.  That is now imported by the testing.